.travis.yml

```diff
@@ -11,6 +11,7 @@ addons:
     packages:
       - fakeroot
       - lintian
+      - fakeroot
```

Comment:
This is duplicate.

Reply:
Good catch, the result of rebasing and `eslint` doesn't check for this